### ISSUE TITLE
Unable to find mixed precision mode switch

### ISSUE THREAD

**Issue by AmemiyaYuko** [onnx/onnx-tensorrt#624]
While I was looking around Nvidia document, I found this https://docs.nvidia.com/deeplearning/frameworks/tf-trt-user-guide/index.html

> converter = trt.TrtGraphConverter(
    input_saved_model_dir=input_saved_model_dir,
    max_workspace_size_bytes=(11<32),
    precision_mode=”FP16”,
    maximum_cached_engines=100)
Could I set precision_mode in onnx-tensorrt?

**Comment by kevinch-nv** [comment]
Yes, we have two tools documented [here ](https://github.com/onnx/onnx-tensorrt/blob/master/docs/faq.md#how-do-i-import-and-run-an-onnx-model-through-tensorrt) that you can use. Typically to change the precision you can use the flags `--fp16` / `--int8` to have TensorRT optimize your networks in those precisions.
